src/Components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../CSS/Header.css'
+import background from '../assets/header-background.jpeg'
+
+const Header = () => {
+	return (
+		<div className="header-container" style={{backgroundImage: `url('https://data.whicdn.com/images/360077678/original.gif')`}}>
```

Comment:
Do you plan to eventually make the backgroundImage dynamic?  If not, I might suggest moving the backgroundImage to a CSS file rather than using inline styling.